### DISCUSSION TITLE
Is there a plan to start supporting image uploads into the API

### DISCUSSION THREAD

**Discussion by MikeyBlueD** [perplexityai/api-discussion#72] [Ideas]
Are there any plans for starting to support image uploads into the API and get the texts, or a description back?

**Comment by shubhang98** [comment]
Yes we will support it in the future but we don't have a specific timeline for it.

**Reply by seacar** [reply]
Any updates here?

**Comment by JoaoCH** [comment]
Hi,

any update on upload image with API?

**Reply by shubhang98** [reply]
Yes, we will have it soon - in the next few weeks.

**Reply by JoaoCH** [reply]
Nice,

It will be vital for the development I am doing.

Will it be possible to obtain what transformations applied to the image before loading it on the model?

For example: For what values is the image resized?

**Reply by JoaoCH** [reply]
> Yes, we will have it soon - in the next few weeks.

Hi,

Is there a ETA on upload image from API.
